docs/validator/economics.md

```diff
@@ -0,0 +1,132 @@
+There are many more slashing conditions, with documentation that is work in progress, please refer to the [EpochManager specs](https://github.com/nearprotocol/NEPs/pull/37) if you want to know more.
+
+
+## 3. Continuous operation
+
+Validators must keep the operations up and running by:
+1. Setting the *Service Level Objectives*
+2. Leveraging market dynamics
+
+### Setting the Service Level Objectives
+
+Achieving 90% of uptime is not sufficient: any new release of the node should be tested and applied without interruptions. Leveraging infrastructure-as-code usually simplifies ongoing maintenance, increasing efficiency and profitability at the cost of higher upfront investments.
+On top of ordinary maintenance, your operations will require an incident/response playbook, to promptly resolve any infrastructure failure, as required by the `MINIMUM_THRESHOLD`.
+
+
+### Leverage market dynamics
+
+NEAR validators are subject to market and game-theory dynamics, relating to:
+1. The total of NEAR tokens at stake
+2. The median of the stake across validators
+3. Protocol fees and inflation
+4. The exchange value of NEAR tokens
+5. The opportunity cost of other PoS tokens
+
+**Total of NEAR tokens at stake**
+NEAR rewards algorithm mint 5% new tokens every year. Therefore, the initial supply of 1 billion NEAR will generate 50 million tokens (5% of 1 billion) in stake rewards.
+This reward is fixed, regardless of the stake:
+- A relatively small stake of 100 million (10% of 1 billion) will generate a nominal return of 50% (50 million NEAR for a stake of 100 million), thus attracting more staking.
+- On the other hand, a large stake of 800 million (80% of 1 billion) will return 6.25% in rewards (50 million NEAR for a stake of 800 million), which may become too low to lock so many tokens.
+
+<blockquote class="info">
+    <strong>did you know?</strong><br><br>
+    The protocol automatically re-stakes your rewards at the end of each epoch, so you may expect up to 5% of additional stake every year.
+</blockquote>
+
+**Median of the stake across validators**
+A high median will have fewer validators with large stakes, thus introducing higher barriers of entry for any new `proposal`. On the other hand, a low median decreases the needed stake, introducing more turnaround and competition between validators.
+You may expect the latter, assuming that delegators will spread their stake across multiple validators to hedge risks.
+
+**Protocol fees and inflation**
+In detail, rewards follow the formula `total_supply * 0.05/730` of NEAR tokens at the end of every `epoch` (producing 5% of inflation per year). However, the protocol *burns* the costs of `tx_fees + state_rent` at the end of each block - such that rewards increase `total_supply` and fees reduce it. 
```

Comment:
I would not use 5% of inflation per year, because later you say that inflation is lower. May be say "producing 5% of new tokens per year".
Also there is a bit of computation weirdness here, because we issue 0.00009433% of total supply every epoch, and as total supply fluctuate, it's not actually going to be for example 5% of total supply on day 1.